src/test/daml/Daml/Finance/Account/Test/Controllers.daml

```diff
@@ -70,22 +86,33 @@ ownerAndCustodianControlled = script do
     setupInitialState tp (Fungible.Factory with provider = custodian; observers = pp) []
       OwnerAndCustodian
 
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [issuer, investor] issuerHoldingCid investorAccount
+
   -- Owners can't transfer.
   submitMultiMustFail [issuer, investor] [publicParty] do
     exerciseCmd (coerceInterfaceContractId @Transferable.I issuerHoldingCid)
       Transferable.Transfer with
-        actors = S.fromList [issuer, investor]; newOwnerAccount = investorAccount
+        actors = S.fromList [issuer, investor]; newOwnerAccount = investorAccount; idx
+
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [custodian] issuerHoldingCid investorAccount
```

Comment:
no need

Reply:
this is done to check proper contract visibility